components/tabs/Reuse/ReuseTabsService.cs

```diff
@@ -176,6 +176,50 @@ private void AddReuseTabsPageItem(Type pageType)
             _pageMap[url] = reuseTabsPageItem;
         }
 
+
+          
+     
+        public void AddReuseTabsPageItem(string pageUrl, string? routeUrl, RenderFragment? title)
+        {
+            /*
+              Use it like this
+                TabService.AddReuseTabsPageItem("/module/1000/1001", "/module/{group}/{menu}", (builder) => { builder.AddContent(0, "Title"); });
+                TabService.AddReuseTabsPageItem("/module/1000/1002", "/module/{group}/{menu}", null);
+                TabService.AddReuseTabsPageItem("/user/setting", null, null);
+             */
+        
+            if (!_pageMap.ContainsKey(pageUrl))
+            {
```

Comment:
Please return immediately while the _pageMap cantains the pageUrl.

Reply:
The above is to judge that it does not exist before entering the loop code, why do you need to judge that there exists and return it directly? Are you trying to add else {return;} } ?

Reply:
I just want to reduce cod block nesting